mkosi/installer/zypper.py

```diff
@@ -126,7 +126,8 @@ def invoke(
         stdout: _FILE = None,
     ) -> CompletedProcess:
         return run(
-            cls.cmd(context) + [operation, *arguments],
+            cls.cmd(context)
+            + [*(["--ignore-unknown"] if operation == "remove" else []), operation, *arguments],
```

Comment:
Please add an `options` argument and pass this in instead of putting it in `invoke()` itself.

Reply:
Sure, changed.